### PR TITLE
fix white text on navbar's problem

### PR DESCRIPTION
**Pull request by barry800414**
add one more class to indicate if the page is full screen image

### CODE REVIEW

**Comment by barry800414** [comment]
don't merge, there is still some problem

**Comment by barry800414** [comment]
Problem solved

**Comment by mark86092** [comment]
能不能有張圖顯示修了的效果

File: css/grayscale.css [hunk 1]
```diff
@@ -59,6 +59,10 @@ a:focus {
     color: #FF5600;
 }
 
+th{
+    text-align: center;
+}
+
```
Comment (mark86092) [inline css/grayscale.css]: 把這個修正新開一個 commit 與 PR，這是不相關的

**Comment by barry800414** [comment]
big screen:
image
image

small screen 
default
default

File: css/grayscale.css (hunk 1 above)
Comment (barry800414) [inline css/grayscale.css]: 我覺得相關阿，就是調整樣式

File: css/grayscale.css [hunk 2]
```diff
@@ -59,9 +59,6 @@ a:focus {
     color: #FF5600;
 }
 
-th{
-    text-align: center;
-}
```
Comment (mark86092) [inline css/grayscale.css]: One line remaining, 你可以看pr上面的 Files Changed https://github.com/goodjoblife/WorkTimeSurvey/pull/28/files

**Comment by mark86092** [comment]
LGTM

**Pull request merged**